src/test/java/org/terasology/flexiblepathfinding/FallingJPSTest.java

```diff
@@ -0,0 +1,184 @@
+// Copyright 2022 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.flexiblepathfinding;
+
+import org.junit.jupiter.api.Test;
+import org.terasology.flexiblepathfinding.helpers.JPSTestHelper;
+import org.terasology.flexiblepathfinding.helpers.TestPaths;
+import org.terasology.flexiblepathfinding.helpers.TestWorlds;
+import org.terasology.flexiblepathfinding.plugins.basic.FallingPlugin;
+
+/**
+ * Falling movements are purely vertical (down) and require additional horizontal walking movements
```

Comment:
The term "movement" or "movement type" is introduce one level higher in Behaviors for the actual physical movement input.

```suggestion
 * Falling describes purely vertical (down) paths and does not allow for horizontal steps.
 *
```